pkg/cmd/kafka/create/data.go

```diff
@@ -0,0 +1,160 @@
+package create
```

Comment:
@wtrocki what do you think about extracting some of this logic to the SDKs when it's finalised so that it is reusable across clients?

Reply:
+1 although I would like to do it once we get some more confidence that this logic is future proof.